src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -420,7 +420,32 @@ module String =
         elif value.Contains(pattern) then
             Some value
         else None
-     
+    
+    open System.IO
+
+    let getLines (str: string) =
+        use reader = new StringReader(str)
+        [|
+        let line = ref (reader.ReadLine())
+        while (!line |> function | null -> false | _ -> true) do
```

Comment:
Comparing to `null` is very inefficient. If I remember properly, @latkin made benchmarks. PM is ok.